### DISCUSSION TITLE
Best practice for tracking resting times

### DISCUSSION THREAD

**Discussion by Mephisztoe** [kimai/kimai#3614] [Questions & Answers]
In Germany employees must take a break of 30min when they work at least 6 hours and up to 9 hours at max. If they work more than 9 hours, they must take a break of 45min. These pauses must take place sometime within working hours. My former companies' time tracking solution allowed for tracking breaks due to legal regulations and even validated the amount set (15, 30, 45, 60 minutes) against the working time tracked for that day.

What is best practices regarding the tracking of breaks using Kimai? 

Initially we thought about configuring an internal project for this. However, it is clumsy and complicate. So at the moment we just skip tracking breaks and only track working times.

For instance:
An employee starts working at 8am. At 12pm, she takes a break for half an hour and then works for another four hours until 4:30pm.
In this case she needs to track two blocks of working hours:
8am to 12pm
12:30pm to 4:30pm

Is this what Kimai "offers" us to do or is there any other "best practice", I am not aware of, yet?

**Comment by kevinpapst** [comment]
Kimai is primarily a project time-tracker.
You seem to look for a software that has features for working time account (Arbeitszeitkonten).
Check out the plugins controlling,vacation,sickness in [the marketplace](https://www.kimai.org/store/).
They are not mine, so i cannot answer questions about those.

In Kimai itself, doing a break means not tracking time.

But you could also track attendance time (e.g. one entry from 8 to 16) and book a second entry for the rest break (probably utilizing the deduction time plugin to make that second booking have a negative duration).

Actually I cannot share any "best practice", this really depends on what you want to achieve...
